APMrover2/mode.cpp

```diff
@@ -102,6 +102,20 @@ void Mode::set_desired_location(const struct Location& destination, float next_l
     }
 }
 
+// set desired location as an offset from the EKF origin in NED frame
+bool Mode::set_desired_location_NED(const Vector3f& destination, float next_leg_bearing_cd)
+{
+    Location destination_ned;
+    // initialise destination to ekf origin
+    if (!ahrs.get_origin(destination_ned)) {
+        return false;
+    }
+    // apply offset
+    location_offset(destination_ned, destination.x, destination.y);
```

Comment:
Does the height need to be corrected?

Reply:
Not really.  At the moment Rover doesn't makes use of the altitude during navigation.

Reply:
Was wondering if that was the case, but wasn't really familiar with Rover :)

There is the idle thought of if it should be included as future proofing, but its such an addition everywhere else in the code it might not be worth the time/effort at the moment.

Reply:
yes, it crossed my mind as well for the same reason..